src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -360,6 +398,45 @@ class WelcomePage extends Disposable {
 				windowOpenable = { workspaceUri: recent.workspace.configPath };
 			}
 
+			const daysOfWeek: string[] = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+			const monthsOfYear: string[] = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+
+			let lastOpened: string;
+			let mtime: any = fs.statSync(fullPath).mtime;
+			let currentDate: Date = new Date;
+			let daysLapsed: number;
+			let delta = Math.abs(mtime - currentDate.getTime()) / 1000;
+
+			let hours = mtime.getHours();
+			let minutes = mtime.getMinutes();
+			let ampm = hours >= 12 ? 'pm' : 'am';
```

Comment:
This isn't locale-friendly. Using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString seems like that'd be a better option.

Reply:
changed to locale friendly